### ISSUE TITLE
[1.18.2] Hang on entering a world, may be due to "Can't get entityTexture"

### ISSUE THREAD

**Issue by Shadowex3** [TeamJM/journeymap#489]
#### JourneyMap Version:
1.18.2-5.9.0 beta2

#### Forge Version:
1.18.2-40.1.73

#### Describe the problem in repeatable steps:

1. Create a world with untamed wilds and/or exotic birds
2. Enter the world
3. Click "back to game" to unpause

#### Relevant errors from `.minecraft/journeymap/journeymap.log`:

[full MC log](https://paste.ee/p/o81No)
[full journeymap log](https://paste.ee/p/kW5v9)

Appears to be the relevant excerpt
```
[15:59:08] [Render thread/ERROR] [journeymap/]: Can't get entityTexture for TranslatableComponent{key='entity.exoticbirds.woodpecker', args=[], siblings=[], style=Style{ color=null, bold=null, italic=null, underlined=null, strikethrough=null, obfuscated=null, clickEvent=null, hoverEvent=null, insertion=null, font=minecraft:default}} (SUPPRESSED)
[15:59:08] [Render thread/WARN] [journeymap/]: java.lang.Throwable: Can't get entityTexture for TranslatableComponent{key='entity.exoticbirds.woodpecker', args=[], siblings=[], style=Style{ color=null, bold=null, italic=null, underlined=null, strikethrough=null, obfuscated=null, clickEvent=null, hoverEvent=null, insertion=null, font=minecraft:default}}
[15:59:08] [Render thread/ERROR] [journeymap/]: Can't get entityTexture for TranslatableComponent{key='entity.untamedwilds.big_cat', args=[], siblings=[], style=Style{ color=null, bold=null, italic=null, underlined=null, strikethrough=null, obfuscated=null, clickEvent=null, hoverEvent=null, insertion=null, font=minecraft:default}} (SUPPRESSED)
[15:59:08] [Render thread/WARN] [journeymap/]: java.lang.Throwable: Can't get entityTexture for TranslatableComponent{key='entity.untamedwilds.big_cat', args=[], siblings=[], style=Style{ color=null, bold=null, italic=null, underlined=null, strikethrough=null, obfuscated=null, clickEvent=null, hoverEvent=null, insertion=null, font=minecraft:default}}
[15:59:08] [Render thread/ERROR] [journeymap/]: Can't get entityTexture for TranslatableComponent{key='entity.untamedwilds.boar', args=[], siblings=[], style=Style{ color=null, bold=null, italic=null, underlined=null, strikethrough=null, obfuscated=null, clickEvent=null, hoverEvent=null, insertion=null, font=minecraft:default}} (SUPPRESSED)
[15:59:08] [Render thread/WARN] [journeymap/]: java.lang.Throwable: Can't get entityTexture for TranslatableComponent{key='entity.untamedwilds.boar', args=[], siblings=[], style=Style{ color=null, bold=null, italic=null, underlined=null, strikethrough=null, obfuscated=null, clickEvent=null, hoverEvent=null, insertion=null, font=minecraft:default}}
[15:59:08] [Render thread/ERROR] [journeymap/]: Can't get entityTexture for TranslatableComponent{key='entity.untamedwilds.rhino', args=[], siblings=[], style=Style{ color=null, bold=null, italic=null, underlined=null, strikethrough=null, obfuscated=null, clickEvent=null, hoverEvent=null, insertion=null, font=minecraft:default}} (SUPPRESSED)
[15:59:08] [Render thread/WARN] [journeymap/]: java.lang.Throwable: Can't get entityTexture for TranslatableComponent{key='entity.untamedwilds.rhino', args=[], siblings=[], style=Style{ color=null, bold=null, italic=null, underlined=null, strikethrough=null, obfuscated=null, clickEvent=null, hoverEvent=null, insertion=null, font=minecraft:default}}
[15:59:08] [Render thread/WARN] [mixin/]: Error loading class: journeymap/client/render/texture/TextureImpl (java.lang.ClassNotFoundException: journeymap.client.render.texture.TextureImpl)
```

After this hits the console minecraft freezes until the process is killed. Trying to play the exact same save/world without journeymap works fine. I thought this might be due to Distant Horizons but I got the same hang without it.

**Comment by mysticdrew** [comment]
Looks like a mod may be trying to use a mixin to modify journeymap
```[15:59:08] [Render thread/WARN] [mixin/]: Error loading class: journeymap/client/render/texture/TextureImpl (java.lang.ClassNotFoundException: journeymap.client.render.texture.TextureImpl)```
Those entityTexture warnings would not cause an issue as it is just a log output for modded mobs that do not have a texture for displaying on the map

**Comment by mysticdrew** [comment]
The issue is with MerchantMarkers-1.18.2-1.2.0

**Comment by Shadowex3** [comment]
Spiffy thank you, I'll dump that one.

**Comment by mysticdrew** [comment]
> Spiffy thank you, I'll dump that one.

I have already informed the MerchantMarker's dev.

**Comment by AHilyard** [comment]
@Shadowex3 Thanks for bringing this to my attention, I've just released Merchant Markers version 1.2.2 that fixes this (https://www.curseforge.com/minecraft/mc-mods/merchant-markers/files/3928334), if you feel like un-dumping at some point.
